### DISCUSSION TITLE
Filter List from Local File

### DISCUSSION THREAD

**Discussion by bnaman50** [darkreader/darkreader#8395] [Ideas]
Hello, 

First of all, let me say that this extension is an eye saver (😉) and works like a charm. 

I have a couple of more suggestions that I feel would be quite helpful. 
1. It should be able to read a local file containing the list of sites to be filtered. 
2. There should be a functionality to maintain separate list for 'Invert' and 'Not Invert' websites. 

I am not sure if these features are already there. I could not find any such thing on the FAQ/Help page. If you are aware of it, please let me know. 

Thanks, 
Naman

**Comment by Gusted** [comment]
> 1. It should be able to read a local file containing the list of sites to be filtered.

Why from a local file? There's already a built-in list that allows you to save it via the browser. As well, browsers don't allow extensions to read from the file system.



> 2\. There should be a functionality to maintain separate list for 'Invert' and 'Not Invert' websites.

#28

**Reply by bnaman50** [reply]
Hey, @Gusted , again, thanks for your quick response. 

> Why from a local file? There's already a built-in list that allows you to save it via the browser. As well, browsers don't allow extensions to read from the file system.

I was hoping I could maintain a local file or it could read the list from some link or something. Also, could you tell me how can I export my list?

>  > There should be a functionality to maintain separate list for 'Invert' and 'Not Invert' websites.

> https://github.com/darkreader/darkreader/issues/28

Thanks for this. 

Naman

**Reply by Gusted** [reply]
> I was hoping I could maintain a local file or it could read the list from some link or something. Also, could you tell me how can I export my list?

#615
